bosh/ruby-app-release/unit-tests/spec/helpers.rb

```diff
@@ -0,0 +1,48 @@
+def check_block_formations(rendered_template)
+
+  line_index = 0
+  tokens = Array.new #acts like a stack
+  exec_line = rendered_template.each_line do |line|
+    if line.include? "case"
+      tokens.push("case")
+    elsif line.match? (/.+\)/)
+      tokens.push("open")
+    elsif line.include? ";;"
+      tokens.push("close")
+    end
+  end
+
+  case_token = tokens.shift
+  return false unless case_token == "case"
+  
+  remaining_tokens_count = tokens.length
+
+  next_token = "close"
+
+  for i in 1..remaining_tokens_count do
+    t = tokens.pop
+
+    return false unless t == next_token
+
+    if next_token == "close"
+      next_token = "open"
+    elsif
+      next_token = "close"
+    end
+  end
+
+  return true
```

Comment:
```suggestion
  true
```

Reply:
Do you think this Ruby-esque style of returning is more readable? I find it very very weird. If it was a statement itself yes, I would leave it in the end without adding `return`, but for this one I find it slightly annoying

Reply:
We never use explicit `return`s in our code, so we do here the same I'd say